packages/react/src/themes/teams/components/Flex/flexVariables.ts

```diff
@@ -8,6 +8,7 @@ export type FlexVariables = GapValues & PaddingValues
 
 export default (): FlexVariables => ({
   // GAP VALUES
+  'gap.list.item': pxToRem(8),
```

Comment:
Would be better to name it `gap.smaller`? Thinking of further other usages.

Reply:
Yes, `gap.smaller` probably makes more sense for now.

***

More context:

We are likely to end up organizing our design system values (variables, etc) similar to the CTI (category, type, item) pattern, as seen in Style Dictionary [here](https://amzn.github.io/style-dictionary/#/properties).  In that case, we'd this would be something like `gap.listItem.small`.

This pattern is great because qualitative terms like "small" are relative.   A "smaller gap" between icons is not the same value as a "smaller gap" between vertical page sections.

However, design system properties that involve component names (i.e. `listItem`) would be defined somewhere at the top of the framework.  We won't define values for a one component inside of another component, as in, we won't define ListItem design system values in the Flex component.

This will soon be my primary focus, working on our specification, design language, and design system structure.

Reply:
I was actually talking with @kuzhelov and he told me that we are planning to add context dependent values for the props here, that's why I added this value, but I would agree that at this moment `gap.smaller` would make more sense. We can update it later. Thanks for the suggestions!